tests/Oryx.RuntimeImage.Tests/DotNetCore/DotnetCoreImageVersionsTest.cs

```diff
@@ -88,6 +88,7 @@ public void RuntimeImage_HasExecptedDotNetVersion_NetCoreApp10Versions(string ve
         [InlineData("2.1", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp21)]
         [InlineData("2.2", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp22)]
         [InlineData("3.0", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp30)]
+        [InlineData("3.1", "Version: " + DotNetCoreRunTimeVersions.NetCoreApp31PreviewName)]
```

Comment:
@kichalla same question here about whether or not we should add a test using `DotNetCore31SdkVersion` here

Reply:
Same here, we depend on the output of `dotnet --info` to verify which still prints out preview name. We will modify this test once the released version is available in December.